<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>payservice</class>
 <widget class="QDialog" name="payservice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="accountComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Счет:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тип услуги:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="serviceComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Оплатить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="amountEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
